apple/imac/14-2/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+{
+  imports = [
+    ../.
+    ../../../common/gpu/intel
+    ../../../common/gpu/nvidia
+    ../../../common/hidpi.nix
```

Comment:
I believe we got rid of this?

```suggestion
```

Reply:
No it's still there.

Reply:
Ah. We just removed quite a bunch of stuff.